agent-bootstrap/src/main/java/io/opentelemetry/auto/typedspan/DbCassandraSpan.java

```diff
@@ -0,0 +1,563 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.opentelemetry.auto.typedspan;
+
+import io.opentelemetry.trace.Span;
+import io.opentelemetry.trace.SpanContext;
+import io.opentelemetry.trace.Tracer;
+import java.util.logging.Logger;
+
+/**
+ * <b>Required attributes:</b>
+ *
+ * <ul>
+ *   <li>db.system: An identifier for the database management system (DBMS) product being used. See
+ *       below for a list of well-known identifiers.
```

Comment:
This `see below` is not quite helpful here :) Probably should even be fixed here, as this is known to be `Cassandra`